### DISCUSSION TITLE
Allow checkmark widget to be resized

### DISCUSSION THREAD

**Discussion by iSoron** [iSoron/uhabits#18] [Feature Requests]
Currently, the checkmark widget has a fixed size of 1x1. It would be nice if it could be resized to display more information. For example, it could display checkmarks for the previous days, similarly to the main app.

**Comment by sh0nuff** [comment]
Surprised this isn't more upvoted, since in Nova and other launchers it's easy to resize widgets, allowing users to create custom layouts with bigger squares and horizontal / vertical bars for certain tasks that might be more important

Edit : this would also help the accessibility support for this app, since there's no way to change the font size in tbe app itself, there could be an option it increase the font size and weight to match the size of the widget
